examples/tensor/static_tensor.cpp

```diff
@@ -64,7 +64,17 @@ int main() {
     
     std::cout<<"( exp2 == exp3 ) => "<<( exp2 == exp3 ? "True" : "False")<<'\n';
 
+    /*--------------------------Assignment------------------------------*/
 
+    // Assigning fixed_rank_tensor into static_tensor
+    s_tensor t4 = t3;
+
+    // Assigning dynamic_tensor into static_tensor
+    s_tensor t5 = dt1;
+
+    std::cout<<"( t4 == t3 ) => "<<( t4 == t3 ? "True" : "False")<<'\n';
+    
+    std::cout<<"( t5 == dt1 ) => "<<( t5 == dt1 ? "True" : "False")<<'\n';
```

Comment:
Same comment as in `examples/tensor/fixed_rank_tensor.cpp`.

Reply:
Same as [here](https://github.com/boostorg/ublas/pull/81#discussion_r413331012).